examples/FsCheck.NUnit.Examples/PropertyExamples.fs

```diff
@@ -10,10 +10,18 @@ type NUnitTest() =
     [<Test>]
     member __.NormalTest() =
         ignore true
-
+        
     [<Property>]
     member __.RevUnit (x:char) =
         List.rev [x] = [x]
+
+    // Parallelism for PropertyAttribute not implemented for Nunit yet
```

Comment:
I did it for Xunit using `Replay` as example. But for Nunit `Replay` isn't implemented, and I have no idea for what reason? Are there any restrictions on Nunit side? Should I implement parallelism and replay there?

Reply:
Probably just because no one did it yet. But let's keep those changes for later PRs; this one is big enough as it is. If you do decide to add Replay for NUnit then please do that on master first; then I'll merge it back here, and after that merge hits the fscheck3 branch we can add parallellism to it.

Reply:
Got it, I will send PR to master.